packages/cli/src/utils/checkMissingAppInfo.js

```diff
@@ -0,0 +1,7 @@
+module.exports = async function (app: Object) {
```

Comment:
The other files in the `utils` directory use `kebab-case` for their file naming. Can you please update this file to match?